pkgs/development/compilers/ghc/7.10.1.nix

```diff
@@ -28,9 +35,14 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  # We patch Cabal for GHCJS. See: https://github.com/haskell/cabal/issues/2454
+  postPatch = ''
+    (
```

Comment:
Won't the use of a bracketed expression here mean that error codes during patching will be lost / ignored by the builder?